contracts/test/Gateway.t.sol

```diff
@@ -749,4 +749,13 @@ contract GatewayTest is Test {
             InboundMessage(bridgeHubParaID, 1, command, params, 1, maxRefund, reward), proof, makeMockProof()
         );
     }
+
+    function testSetTokenFees() public {
+        GatewayMock(address(gateway)).setTokenTransferFeesPublic(
+            abi.encode(Gateway.SetTokenTransferFeesParams({register: 1, send: 1}))
```

Comment:
The default fees are already (1, 1), so this test could pass even if the command failed somehow.

Reply:
Actually the default are 1(ether) not 1(wei), anyway check fee before and after set and make sure change as expected.
https://github.com/Snowfork/snowbridge/pull/991/commits/0a2cb1682245675f5189147702a126583eccc1c2